packages/contracts-bridge/contracts/BridgeRouter.sol

```diff
@@ -225,6 +244,72 @@ contract BridgeRouter is Version0, Router {
         IBridgeToken(_currentRepr).mint(msg.sender, _bal);
     }
 
+    // ============ Internal: Send ============
+
+    /**
+     * @notice Debit tokens from the function caller
+     * as part of sending tokens across chains
+     * @dev Locks canonical tokens in escrow in BridgeRouter
+     * OR Burns representation tokens
+     * @param _token The token to pull from the sender
+     * @param _amount The amount to pull from the sender
+     * @return _tokenId the bytes canonical token identifier
+     * @return _detailsHash the hash of the canonical token details (name, symbol, decimal)
+     */
+    function _debitTokens(address _token, uint256 _amount)
```

Comment:
this is a non-functional refactor, and should be in a separate PR before this one.

Reply:
https://github.com/nomad-xyz/monorepo/pull/385